uptane/clients/primary.py

```diff
@@ -243,7 +243,8 @@ def __init__(
 
       timeserver_public_key See class docstring above.
 
-      my_secondaries        See class docstring above. (optional)
+      my_secondaries        See class docstring above. (optional: will also be
+                            populated by communication from Secondaries)
```

Comment:
I don't quite follow this comment.  Is there a method that secondaries can call to modify `self.my_secondaries`?

The comment might be clearer if you stated how `my_secondaries` may be updated/populated.